delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/identity/IdentitiesImpl.scala

```diff
@@ -77,7 +81,7 @@ class IdentitiesImpl private (findActiveRealm: String => UIO[Option[Realm]], gro
       parsedToken       <- IO.fromEither(ParsedToken.fromToken(token))
       activeRealmOption <- findActiveRealm(parsedToken.issuer)
       activeRealm       <- IO.fromOption(activeRealmOption, UnknownAccessTokenIssuer)
-      _                 <- validate(parsedToken.jwtToken, realmKeyset(activeRealm))
+      _                 <- validate(activeRealm.acceptedAudiences, parsedToken.jwtToken, realmKeyset(activeRealm))
```

Comment:
We should update`InvalidAccessToken` message to mention audiences as an error cause